cypress/e2e/Test_F1.cy.js

```diff
@@ -0,0 +1,117 @@
+describe('Login tests!!!', () => {
+
+  it('ID = 1AC1 ', () => {
+    cy.visit('/').url().should('eq', 'https://www.saucedemo.com/')
+    cy.get('[data-test=username').type("standard_user")
+    cy.get('[data-test=password').type("secret_sauce")
+    cy.get('[data-test=login-button').click()
+    cy.get('[data-test=product_sort_container]').should('be.visible')
+  })
```

Comment:
general comment - verify if cookie is properly set up after clicking on login button. use cy.getCookie()